include/flatbuffers/code_generators.h

```diff
@@ -18,6 +18,9 @@
 #define FLATBUFFERS_CODE_GENERATORS_H_
 
 namespace flatbuffers {
+static const char *g_flatbuffers_warning =
```

Comment:
This may end up in all compilation units.. instead make a `BaseGenerator::FlatBuffersGeneratedWarning()` that returns the string?